apps.js

```diff
@@ -76,6 +78,8 @@ function createApp(type, addRoutes) {
     stream: logger.stream
   }));
   app.use(middleware.requireHttps(cfg));
+  app.use(connectTimeout(CONST.HTTP_TIMEOUT.TIMEOUT_DURATION));
```

Comment:
pass second parameter options with `respond` set to true. By setting it, you dont need the next middleware and can move the code that is in middleware.timeout to the global error handler in middleware.